server/auth/types/claims.go

```diff
@@ -12,7 +12,7 @@ type Claims struct {
 	jwt.Claims
 	Groups                  []string               `json:"groups,omitempty"`
 	Email                   string                 `json:"email,omitempty"`
-	EmailVerified           bool                   `json:"email_verified,omitempty"`
+	EmailVerified           bool                   `json:"-"`
```

Comment:
Hmm I'm thinking the more correct way of handling this would have been to use a custom type like in [this SO question's answers](https://stackoverflow.com/q/30856454/3431180)